Test.c

```diff
@@ -4,11 +4,151 @@
 // Ayisha Sameera,GET.
 // ------------------------------------------------------------------
 // Test.c
-// Program on branch main
+// Program on branch A5
 // ------------------------------------------------------------------------------------------------
-
+#pragma warning(disable:4996)
+#include "Header.h"
 #include <stdio.h>
+#include <stdbool.h>
+#include <malloc.h>
+#include <stdlib.h>
+#include <limits.h>
+
+#define MAX_SIZE 20
+
+typedef struct {
+   int Array[MAX_SIZE];
+   int Length;
+}Test;
+
+/// <summary>Checks if the array is sorted in ascending order.</summary>
+bool IsSorted (int arr[], int length) {
+   for (int i = 1; i < length; i++) if (arr[i - 1] > arr[i]) return false;
+   return true;
+}
+
+void PrintArray (int arr[], int size, int expLength) {

```

Comment:
Since you're working with arrays of constant length, you can directly use 8 here (or declare it as a local variable.